homeassistant/components/feedreader/coordinator.py

```diff
@@ -101,7 +102,11 @@ def _parse_feed() -> feedparser.FeedParserDict:
 
     async def async_setup(self) -> None:
```

Comment:
Consider renaming this function `_async_setup`. It is run during `async_config_entry_first_refresh` and does the same kind of error handling as `_async_update_data`

Reply:
Exactly what I was writing, and then move up the `async_config_entry_first_refresh()` call in `__init__.py`

Reply:
wasn't aware that we have a `_async_setup` - is this new?
will rework it ...

Reply:
Month or 2-3 old

Reply:
Time goes fast... At least 3 months's I think...

Reply:
ahhh ... it was on purpose to split the setup from the first data fetch - see

https://github.com/home-assistant/core/blob/c23f5c9f2c313d67d1bf417ff8a656ffadabc75f/homeassistant/components/feedreader/__init__.py#L33-L41

Reply:
> Time goes fast... At least 3 months's I think... **Update** 5 months actually

i really need to subscribe to the dev blog ... aaaand read it :grimacing:

Reply:
Did you intend to fix it here, as it's still not

Reply:
> Did you intend to fix it here, as it's still not

nope, as written in https://github.com/home-assistant/core/pull/134202#discussion_r1899132554 we can't combine the setup and data fetch, since we need to setup the `event` platform first

Reply:
That seems weird but ok